packages/components/button/src/Button.stories.tsx

```diff
@@ -103,10 +105,44 @@ export const Icons: StoryFn = _args => (
   </div>
 )
 
+export const Loading: StoryFn = () => {
+  const [isLoading, setIsLoading] = useState(true)
+
+  return (
+    <div className="gap-lg flex flex-col">
+      <Checkbox checked={isLoading} onClick={() => setIsLoading(!isLoading)}>
```

Comment:
This is a personal opinion but I would try to keep the stories as simple as possible and make then atomic. This way developers can easily copy/paste and our visual regression tests would be more granular